next-steps.md

```diff
@@ -2,18 +2,18 @@
 
 Even though we're nowhere near done collecting data, events are pushing us to think about what we do next. We're already seeing changes to government websites, and we need to be able to:
 
-1. [monitor websites](#monitoring-websites)
-1. [filter changes](#filtering-changes)
-1. [audit resiliency](#auditing-resiliency)
-1. [~~understand data~~](#understanding-data)
-1. [disseminating](#disseminating)
+1. [monitor websites](#monitoring-websites) (work with @titaniumbones in the [pagefreezer-cli](https://github.com/edgi-govdata-archiving/pagefreezer-cli) repo)
+1. [filter changes](#filtering-changes) (work with @titaniumbones in the [pagefreezer-cli](https://github.com/edgi-govdata-archiving/pagefreezer-cli) repo)
+1. [audit resiliency](#auditing-resiliency) (work with _@librlaurie (??) or @dcwalk (??)_ in _Google Docs??_)
```

Comment:
Yeah, hard ot say I guess. Lots of people in #data-archiving, no one that I can tell really signed up for other tracks.